packages/patternfly-3/patternfly-react/package.json

```diff
@@ -27,6 +27,7 @@
     "bootstrap-slider-without-jquery": "^10.0.0",
     "breakjs": "^1.0.0",
     "classnames": "^2.2.5",
+    "copy-to-clipboard": "^3.0.8",
```

Comment:
Thanks @seanforyou23 ! You're probably right,

I used it because it supports many browsers and I thought it would help me concentrate on the implementation of the component.

I will address these things in the following weeks. :smile:

Reply:
No problem @boaz1337 , it's always good to progress incrementally so thank you for doing the hard part of getting the ball rolling with this! Feel free to reach out if you run into issues/questions - I'm here to help!